mmdet/datasets/xml_style.py

```diff
@@ -18,11 +18,19 @@ class XMLDataset(CustomDataset):
         min_size (int | float, optional): The minimum size of bounding
             boxes in the images. If the size of a bounding box is less than
             ``min_size``, it would be add to ignored field.
+        img_subdir (str): Subdir where images are stored. Default is JPEGImages.
```

Comment:
1. can we use `img_prefix` rather than `img_subdir` to do what we want through setting proper values to `img_prefix`?
2. if we can do 1, considering rename ann_subdir -> ann_prefix

Reply:
We could... but then, img_prefix will be behaving differently than before...

A reimplementation for this class could be done. but my purpose for this PR was just removing the hardcoded parts, so it could be configured, as i had to do this for using it with another dataset.

Let me think about it...

Reply:
@ZwwWayne if you think it's useful we can have a look at a little refactor on that part for later on.